packages/core/src/presets.ts

```diff
@@ -148,3 +148,11 @@ export interface CoolifyEnv {
   PORT?: string;
   HOST?: string;
 }
+
+export interface ViteEnv {
+  BASE_URL?: string;
+  MODE?: string;
+  DEV?: boolean;
+  PROD?: boolean;
+  SSR?: boolean;
+}
```

Comment:
do these need to be optional?

Reply:
Initially I thought, yes. These aren't required for us to be defined anywhere in a Vite app to run.

Now that you have asked, I gave it another thought and now realize that these shouldn't be optional. These always exists from Vite side. There are always default value from Vite for all these.

Updated.